donald_wade/week_04/tuesday/mta3.rb

```diff
@@ -0,0 +1,48 @@
+require "pp"
+
+subway = {
+  n: ["Times Square", "34th Street", "28th Street", "23rd Street", "Union Square", "8th Avenue"],
+
+  l: ["8th Avenue", "6th Avenue", "Union Square", "3rd Street", "1st Avenue"],
+
+  l6: ["Grand Central", "33rd Street", "28th Street", "23rd Street", "Union Square", "Astor Place"],
+}
+
+def get_to_station(on, off, myline, subway)
```

Comment:
You don't need to pass subway in here, as in your program subway is a global variable and you can access it in any method you have here.